### DISCUSSION TITLE
List of cameras successfully working?

### DISCUSSION THREAD

**Discussion by judasferret** [QuantumEntangledAndy/neolink#207] [General]
Is there a list of cameras successfully working with this codebase? I'm about to plunge in and buy a bunch of reolink cameras so was hoping to find a list but have not yet.

If there isn't a list, reply back with the model(s) of camera you have in your successful setup!

If you're keen also reply with what hardware you host neolink on and how many cameras are in your setup. 

Tested Camera Models (know installations) 
- B800/D800 (1)
- B400/D400 (1)
- E1 (1)
- Lumus (1)
- 510A (1)
- Argus3 (1)
- Argus 3 Ultra (not working) 
- Argus2 Eco (1)
- Argus PT (1)
- Argus Eco (1)
- RLC 410 (1)
- RLC-423ws (1)
- RLC 810a (1)
- Swann NHD-815 (sub stream only) 

Tested Hosts (know installations) 
- RaspPi 3B (1)

Sources of others not in this discussion:
[Argus PT - Elhennig](https://github.com/QuantumEntangledAndy/neolink/discussions/76#discussion-5136952)
[Argus Eco ianK6449](https://github.com/QuantumEntangledAndy/neolink/discussions/80#discussion-5173668) 
[RLC 810a](https://github.com/QuantumEntangledAndy/neolink/issues/201#issuecomment-1848971199) 
[RLC-423ws](https://github.com/QuantumEntangledAndy/neolink/issues/188#issue-1992174809)
[Swann NHD-815](https://github.com/QuantumEntangledAndy/neolink/issues/191#issue-1997972082)
[Argus 3 Ultra - not working 25/01/2024](https://github.com/QuantumEntangledAndy/neolink/issues/182#issue-1974865312)

**Comment by QuantumEntangledAndy** [comment]
Nothing formal but I know the following

- B800/D800
- B400/D400
- E1
- Lumus
- 510A
- Argus3
- Argus2 Eco

**Reply by QuantumEntangledAndy** [reply]
Most hosts really, but assuming your looking at the smaller boards you can add

RaspPi 3B

My Rpi tests are run on this
